notebooks/250-fastcomposer-image-generation/250-fastcomposer-image-generation.ipynb

```diff
@@ -0,0 +1,1352 @@
+{
```

Comment:
<p>could you please give more explanation about what happens here e.g. demonstrate input images of personalities that will be a part of generated image</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
More description is added